dashboards/jupyterhub.jsonnet

```diff
@@ -147,6 +147,29 @@ local hubResponseLatency = graphPanel.new(
 local allComponentsMemory = jupyterhub.memoryPanel('All JupyterHub Components', component='singleuser-server', multi=true);
 local allComponentsCPU = jupyterhub.cpuPanel('All JupyterHub Components', component='singleuser-server', multi=true);
 
+local hubDBUsage = graphPanel.new(
+  'Hub DB Disk Space Availability %',
+  description=|||
+    % of disk space left in the disk storing the JupyterHub sqlite database. If goes to 0, the hub will fail.
+  |||,
+  decimals=0,
+  min=0,
+  max=1,
+  formatY1='percentunit',
+  datasource='$PROMETHEUS_DS'
+).addTarget(
+  prometheus.target(
+    |||
+      # Free bytes available on the hub db PVC
+      sum(kubelet_volume_stats_available_bytes{persistentvolumeclaim="hub-db-dir", namespace=~"$hub"}) by (namespace) /
+      # Total number of bytes available on the hub db PVC
+      sum(kubelet_volume_stats_capacity_bytes{persistentvolumeclaim="hub-db-dir", namespace=~"$hub"}) by (namespace)
```

Comment:
Is the pvc name always `hub-db-dir`? From what I get from

https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/ff8fb75cb2a213caf517df3d4885748c1521997c/jupyterhub/templates/_helpers-names.tpl#L105-L108 it depends on `jupyterhub.hub.fullname`?

If this is the default and there's rarely a `jupyterhub.hub.fullname` defined, then I believe it is ok to have this graph be based on it 👍🏼

Reply:
@GeorgianaElena yep, the default is hub-db-dir, and while the fullnameoverride logic is often too complex for me to follow, I think this is good to go for the most part.